https://bugs.gentoo.org/show_bug.cgi?id=186424

"Forgetting" -lm is a common mistake since glibc works without it. Other libc's
are not as forgiving.

--- ./wiretap/configure.in.orig	2007-07-24 13:38:56.000000000 +0400
+++ ./wiretap/configure.in	2007-07-24 13:50:08.000000000 +0400
@@ -324,6 +324,10 @@
 ])
 AC_WIRETAP_PCAP_CHECK
 
+# C libraries that are not libc (like ulibc) requre -lm. bug #1704
+AC_CHECK_LIBM
+AC_SUBST(LIBM)
+
 dnl zlib check
 AC_MSG_CHECKING(whether to use zlib for reading compressed capture files)
 
--- ./wiretap/Makefile.am.orig	2007-07-24 13:38:46.000000000 +0400
+++ ./wiretap/Makefile.am	2007-07-24 13:43:10.000000000 +0400
@@ -69,8 +69,8 @@
 	$(GENERATOR_SOURCES) 	\
 	$(GENERATED_FILES)
 
-libwiretap_la_LIBADD = libwiretap_generated.la
-libwiretap_la_DEPENDENCIES = libwiretap_generated.la
+libwiretap_la_LIBADD = libwiretap_generated.la @LIBM@
+libwiretap_la_DEPENDENCIES = libwiretap_generated.la @LIBM@
 
 ascend-grammar.c : ascend-grammar.y
 	@if [ -z "$(YACC)" ]; then \
